DB/src/main/java/io/deephaven/db/v2/select/AbstractConditionFilter.java

```diff
@@ -152,13 +147,50 @@ public synchronized void init(TableDefinition tableDefinition) {
             }
             params = paramsList.toArray(Param.ZERO_LENGTH_PARAM_ARRAY);
 
+            // check if this is a filter that uses a numba vectorized function
+            for (Param param : params) {
```

Comment:
IMO it's misleading to write this as a loop. It's meant to be an error check followed by some logic, so let's write it like that:

```
            long ncwCount = Arrays.stream(params).filter(p -> p.getValue() instanceof NumbaCallableWrapper).count();
            if (ncwCount != 0) {
                if (params.length != 1) {
                    throw new UncheckedDeephavenException("internal error - misuse of numba vectorized functions wasn't detected.");
                }

                NumbaCallableWrapper numbaCallableWrapper = (NumbaCallableWrapper) params[0].getValue();
```